zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
+import requests
+import json
+import socket
+
+class JoinmeHandler(object):
+    def usage(self) -> str:
+        return '''
+         A bot that will post a link to joinme call. Mention the bot and it will give out instructions.
+        '''
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        if 'ZISSHUDBUNIq' not in message['content']:
+            payload = {'client_id': 'fvg6qhpans4z2jryq45crb69', 'scope': 'user_info scheduler start_meeting',

```

Comment:
This bot shouldn't be tailored to your join.me account. Rather, it should allow anyone to specify their own join.me credentials to use this bot. For that purpose, we have the convention of adding `<bot>.conf` template configuration files for a bot, where a user specifies all the information relevenat to authentication and authorization (most often this is an api key). You can look at the giphy bot as an example on how to do this.